config.js

```diff
@@ -20,6 +20,8 @@
     influxdb: {
       host: host,
       port: port,
+      version: process.env.INFLUXDB_VERSION || "0.8",
+      ssl: process.env.INFLUXDB_SSL || "false",
```

Comment:
I just tested this PR and it works great for me except for the line that sets the `ssl` value. It needs to look like the following code instead so the final value is a boolean rather than a string.

```
      ssl: (process.env.INFLUXDB_SSL || "false").toLowerCase() === "true",
```